src/main.rs

```diff
@@ -25,6 +25,7 @@ fn main() -> Result<(), ExitFailure> {
 
   let dir = args.dir()?;
   let name = args.name()?;
+  let lib = args.lib().unwrap_or(false);
```

Comment:
If `args.lib()` returns a `bool`, we can remove `.unwrap_or(false)` from this line.

Reply:
I opted for this so that it's more clear in `main.rs` what the default vaulue is (instead of hiding it in `cli.rs`. We can of course change this

Reply:
Ah okay, gotcha. Perhaps a better way to be explicit about the default state is to add `default_value=false` to the structopt attribute. I was a bit confused why we had 3 states for a boolean value (e.g. not passed at all, passed as true, passed as false?) instead of defaulting to a "is this option enabled? yes / no"